_data/links.yml

```
@@ -121,10 +121,12 @@
     
   - name: Map EPITA
     icon: fas fa-user-friends
-    href: http://map.epita.eu/
+    href: https://epimap.fr/
     class: tile-small-wide text-dark
     style:
-    - 'background-color: #fff'
+    - 'background-color: #230071'
+    - 'background-image: url(https://epimap.fr/img/logo-min.png)'
+    - 'background-size: cover'
```

I found a better logo for use here, and I fixed it visually.
```suggestion
    - 'background-image: url(https://epimap.fr/img/logo-epimap.svg)'
    - 'background-size: 80%'
    - 'background-position: 20% 20%'
```